the-team/index.html

```diff
@@ -37,771 +37,778 @@
-            <a target="_blank" rel="noopener" href="http://blog.hypesystem.dk">Blog</a>
-        </div>
-    </div>
-    <div class="person-image person-lg-col-4 person-md-col-3 person-sm-col-2 person-xs-col-2">
-        <img src="/img/people/anders-dream-3-sq.jpg" class="dream">
-        <img src="/img/people/anders.jpg">
-    </div>
-    <div
-        class="person person-lg-col-3 person-lg-scoot-right person-md-col-2 person-md-scoot-right person-sm-col-1 person-sm-scoot-right person-xs-scoot-right person-xs-col-1">
-        <div class="person-name">
-            <h2>Anders Enghøj</h2>
-            <div class="person-title">Staff Software Developer. BA Web Development.</div>
-        </div>
-        <div class="person-about">
-            <p>
-                Anders enjoys collaborating with customers in order to find and build the best solution for them.
-                Working in a team and having a mutual goal is what drives Anders in the day-to-day.
-            </p>
-            <p>
-                He enjoys optimizing the work process and when people with different competencies come together to
-                create new and improved solutions.
-                He's also always developing himself and his skills.
-            </p>
+            background: rgb(44, 62, 80);
+            background-image: linear-gradient(rgb(44, 62, 80) 40%, rgb(27, 43, 58));
+            color: #eaf1f7;
+            grid-column: span 2;
+            grid-row: span 2;
+            font-size: 1.5em;
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: center;
+            padding: 100px;
+        }
+        .join-us-container h1 {
+            font-size: 2.5em;
+            font-weight: 100;
+        }
+        .join-us-container p {
+            line-height: 1.5;
+        }
+        @media (min-width: 1350px) and (max-width: 1599px) {
+            .join-us-container {
+                font-size: 1em;
+                grid-column: span 1;
+            }
+        }
+        @media (max-width: 1349px) {
+            .join-us-container {
+                font-size: 1em;
+                grid-row: span 1;
+            }
+        }
+        @media (max-width: 599px) {
+            .join-us-container {
+                grid-column: span 1;
+            }
+        }
+        @media (max-width: 450px) {
+            .join-us-container {
+                padding: 50px;
+            }
+        }
+        @media (max-width: 350px) {
+            .join-us-container {
+                padding: 25px;
+            }
+        }
+    </style>
+    <section class="people-container">
+        <div class="person-image person-lg-col-1 person-md-col-1 person-sm-col-1">
+            <img src="/img/people/niels-dream-2-sq.jpg" class="dream" alt="Abstract variation of Nielse's photo">
+            <img src="/img/people/niels-abildgaard.jpg" alt="Photo of Niels">
```

Comment:
Alt texts on images should never have "Photo" or "Image" in them. Screen readers already read them as images.
If the name is already there in main text, the images can be considered decorative and have `alt=""`